coding-standard-baseline/README.md

```diff
@@ -0,0 +1,34 @@
+# Magento 2 Coding Standard Action
+
+A Github Action that runs the Magento Coding Standard.
+
+## Inputs
+
+See the [action.yml](./action.yml)
+
+## Usage
+
+```yml
+name: Coding Standard baseline
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  coding-standard:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: mage-os/github-actions/coding-standard-baseline@main
+          php_version: 8.1 # Optional, will be used for Php version
+          composer_version: 2
```

Comment:
Given that these are strings, it's probably a good idea to use the string quotes to be clear that they're not numeric. especially for things like composer.